modules/local/bam_rename.nf

```diff
@@ -3,6 +3,11 @@ include { saveFiles; getProcessName } from './functions'
 
 process BAM_RENAME {
     tag "$meta.id"
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/ubuntu:20.04"
```

Comment:
I believe there is a mini version or something that is preferred, as a whole ubuntu installation is huge.

This one: https://github.com/nf-core/modules/blob/20d8250d9f39ddb05dfb437603aaf99b5c0b2b41/modules/gunzip/main.nf#L7-L8

Reply:
Thank you! I have just changed it

Reply:
Sorry, I didn't mind copy-vertabim, I meant use the links.

If you want t copy vertabim:

https://github.com/nf-core/modules/blob/d30bf235b1524327703591c9ce2f372bfb2d474f/modules/gunzip/main.nf#L14-L19

Reply:
Thank you for the clarification! I have just copied and pasted it.